### PR TITLE
komit

### PR DESCRIPTION
**Pull request by slavapopkov**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2004-2014 Fabien Potencier
```
Comment (EduardTrayan) [inline LICENSE]: Please remove this unuseful file

File: UPGRADE-2.2.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+UPGRADE FROM 2.1 to 2.2
```
Comment (EduardTrayan) [inline UPGRADE-2.2.md]: Please remove this unuseful file

File: UPGRADE-2.3.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+UPGRADE FROM 2.2 to 2.3
```
Comment (EduardTrayan) [inline UPGRADE-2.3.md]: Please remove this unuseful file

File: UPGRADE-2.4.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+UPGRADE FROM 2.3 to 2.4
```
Comment (EduardTrayan) [inline UPGRADE-2.4.md]: Please remove this unuseful file

File: UPGRADE.md [hunk 5]
```diff
@@ -0,0 +1,268 @@
+Symfony Standard Edition Upgrade
```
Comment (EduardTrayan) [inline UPGRADE.md]: Please remove this unuseful file

File: app/.htaccess [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<IfModule mod_authz_core.c>
```
Comment (EduardTrayan) [inline app/.htaccess]: Please remove this file if you don't use `apache` as web server

File: src/.htaccess [hunk 7]
```diff
@@ -0,0 +1,7 @@
+<IfModule mod_authz_core.c>
```
Comment (EduardTrayan) [inline src/.htaccess]: Please remove this file if you don't use `apache` as web server

File: src/Acme/DemoBundle/AcmeDemoBundle.php [hunk 8]
```diff
@@ -0,0 +1,9 @@
+<?php
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/AcmeDemoBundle.php]: Please remove standard bundle

File: src/Acme/DemoBundle/Command/HelloWorldCommand.php [hunk 9]
```diff
@@ -0,0 +1,48 @@
+<?php
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Command/HelloWorldCommand.php]: Please remove standard bundle

File: src/Acme/DemoBundle/Controller/DemoController.php [hunk 10]
```diff
@@ -0,0 +1,56 @@
+<?php
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Controller/DemoController.php]: Please remove standard bundle

File: src/Acme/DemoBundle/Controller/SecuredController.php [hunk 11]
```diff
@@ -0,0 +1,70 @@
+<?php
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Controller/SecuredController.php]: Please remove standard bundle

File: src/Acme/DemoBundle/Controller/WelcomeController.php [hunk 12]
```diff
@@ -0,0 +1,19 @@
+<?php
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Controller/WelcomeController.php]: Please remove standard bundle

File: src/Acme/DemoBundle/DependencyInjection/AcmeDemoExtension.php [hunk 13]
```diff
@@ -0,0 +1,22 @@
+<?php
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/DependencyInjection/AcmeDemoExtension.php]: Please remove standard bundle

File: src/Acme/DemoBundle/Form/ContactType.php [hunk 14]
```diff
@@ -0,0 +1,20 @@
+<?php
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Form/ContactType.php]: Please remove standard bundle

File: src/Acme/DemoBundle/Resources/config/routing.yml [hunk 15]
```diff
@@ -0,0 +1,12 @@
+_welcome:
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Resources/config/routing.yml]: Please remove standard bundle

File: src/Acme/DemoBundle/Resources/config/services.xml [hunk 16]
```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" ?>
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Resources/config/services.xml]: Please remove standard bundle

File: src/Acme/DemoBundle/Resources/public/css/demo.css [hunk 17]
```diff
@@ -0,0 +1,101 @@
+body {
```
Comment (EduardTrayan) [inline src/Acme/DemoBundle/Resources/public/css/demo.css]: Please remove standard bundle

File: web/.htaccess [hunk 18]
```diff
@@ -0,0 +1,56 @@
+# Use the front controller as index file. It serves as a fallback solution when
```
Comment (EduardTrayan) [inline web/.htaccess]: Please remove this file if you don't use `apache` as web server

File: LICENSE (hunk 1 above)
Comment (slavapopkov) [inline LICENSE]: оК

File: UPGRADE-2.2.md (hunk 2 above)
Comment (slavapopkov) [inline UPGRADE-2.2.md]: ОК

File: UPGRADE-2.3.md (hunk 3 above)
Comment (slavapopkov) [inline UPGRADE-2.3.md]: ок

File: UPGRADE-2.4.md (hunk 4 above)
Comment (slavapopkov) [inline UPGRADE-2.4.md]: оК

File: UPGRADE.md (hunk 5 above)
Comment (slavapopkov) [inline UPGRADE.md]: оК

File: app/.htaccess (hunk 6 above)
Comment (slavapopkov) [inline app/.htaccess]: Ок

File: src/.htaccess (hunk 7 above)
Comment (slavapopkov) [inline src/.htaccess]: Ок

File: src/Acme/DemoBundle/AcmeDemoBundle.php (hunk 8 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/AcmeDemoBundle.php]: ок

File: src/Acme/DemoBundle/Command/HelloWorldCommand.php (hunk 9 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/Command/HelloWorldCommand.php]: ок

File: src/Acme/DemoBundle/Controller/DemoController.php (hunk 10 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/Controller/DemoController.php]: ок

File: src/Acme/DemoBundle/Controller/SecuredController.php (hunk 11 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/Controller/SecuredController.php]: ок

File: src/Acme/DemoBundle/Controller/WelcomeController.php (hunk 12 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/Controller/WelcomeController.php]: ок

File: src/Acme/DemoBundle/DependencyInjection/AcmeDemoExtension.php (hunk 13 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/DependencyInjection/AcmeDemoExtension.php]: ок

File: src/Acme/DemoBundle/Form/ContactType.php (hunk 14 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/Form/ContactType.php]: ок

File: src/Acme/DemoBundle/Resources/config/services.xml (hunk 16 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/Resources/config/services.xml]: ок

File: src/Acme/DemoBundle/Resources/public/css/demo.css (hunk 17 above)
Comment (slavapopkov) [inline src/Acme/DemoBundle/Resources/public/css/demo.css]: ок

File: web/.htaccess (hunk 18 above)
Comment (slavapopkov) [inline web/.htaccess]: ок

File: app/config/routing_dev.yml [hunk 19]
```diff
@@ -0,0 +1,16 @@
...
+    prefix:   /_profiler
+
+_configurator:
+    resource: "@SensioDistributionBundle/Resources/config/routing/webconfigurator.xml"
+    prefix:   /_configurator
+
+_main:
+    resource: routing.yml
+
+# AcmeDemoBundle routes (to be removed)
```
Comment (EduardTrayan) [inline app/config/routing_dev.yml]: Please remove this commented line

Comment (slavapopkov) [inline app/config/routing_dev.yml]: ок

Comment (EduardTrayan) [inline app/config/routing_dev.yml]: I still see this line

Comment (slavapopkov) [inline app/config/routing_dev.yml]: I delete it

File: app/config/routing.yml [hunk 20]
```diff
@@ -0,0 +1,8 @@
+website:
+    resource: "@WebsiteBundle/Resources/config/routing.yml"
+    prefix:   /
+
+common:
+    resource: "@CommonBundle/Resources/config/routing.yml"
+    prefix:   /
+
```
Comment (EduardTrayan) [inline app/config/routing.yml]: Please remove blank line

File: app/config/routing_dev.yml [hunk 21]
```diff
@@ -0,0 +1,16 @@
...
+    resource: "@WebProfilerBundle/Resources/config/routing/profiler.xml"
+    prefix:   /_profiler
+
+_configurator:
+    resource: "@SensioDistributionBundle/Resources/config/routing/webconfigurator.xml"
+    prefix:   /_configurator
+
+_main:
+    resource: routing.yml
+
```
Comment (EduardTrayan) [inline app/config/routing_dev.yml]: Please remove 2 blank lines

File: app/config/security.yml [hunk 22]
```diff
@@ -0,0 +1,52 @@
...
+                target: _demo
+            #anonymous: ~
+            #http_basic:
+            #    realm: "Secured Demo Area"
+
+    # with these settings you can restrict or allow access for different parts
+    # of your application based on roles, ip, host or methods
+    # http://symfony.com/doc/current/book/security.html#security-book-access-control-matching-options
+    access_control:
+        #- { path: ^/login, roles: IS_AUTHENTICATED_ANONYMOUSLY, requires_channel: https }
```
Comment (EduardTrayan) [inline app/config/security.yml:52]: Please add new line

File: nbproject/private/private.properties [hunk 23]
```diff
@@ -0,0 +1,2 @@
+index.file=index.php
```
Comment (EduardTrayan) [inline nbproject/private/private.properties:1]: Please add your IDE folder `nbproject` to `.gitignore`

File: app/config/routing.yml (hunk 20 above)
Comment (slavapopkov) [inline app/config/routing.yml]: Remove  4, 8 blank

File: app/config/routing_dev.yml (hunk 21 above)
Comment (slavapopkov) [inline app/config/routing_dev.yml]: Ок

File: nbproject/private/private.properties (hunk 23 above)
Comment (slavapopkov) [inline nbproject/private/private.properties:1]: I added  to  .gitignore       +index.file=index.php

File: app/config/security.yml (hunk 22 above)
Comment (slavapopkov) [inline app/config/security.yml:52]: Whehe will I do it-?

**Pull request closed without merging**